### PR TITLE
Add classes

### PR DESCRIPTION
**Pull request by zieeco**
# Awesome Books: Refactor to use classes
**In this milestone, I and my partner achieved the following**
- Refactored the awesome book to use classes in the main.js file
- Ensured there were no linters error

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -3,62 +3,83 @@ const addBookBtn = document.getElementById('add-btn');
 
 let books = [];
```
Comment (Georjane) [inline main.js:4]: - [ ] Can you please initialize this books array inside your class books. For example
```
class Books {
  constructor() {
    this.books = [];
  }
...
}
```
Remember this is a project requirement
image

**Review by Georjane — CHANGES_REQUESTED** [review]
### Hello, Team @zieeco and @Kingstalux, ✋🏽 

Good job so far! Your project works perfectly. You have a very professional Readme file, your PR is professional and you have a good mastery of git-flow. Your linters are all passing, I applaud that too.
However, we have a few more changes to make in order to meet all requirements for this project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding! 💻 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 🤝 

_Code Reviewer
**[Jane](https://witahgeorjane.tech/)**_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: main.js (hunk 1 above)
Comment (Kingstalux) [reply main.js]: Okay thanks, we've implemented that.

Comment (rloterh) [reply main.js]: This issue is well resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Isaac, 

Your project is complete! Great job for making the necessary changes recommended by the previous reviewer(s). There is nothing else to say other than... it's time to merge it 🍾:shipit:.
Congratulations! 🎉🥂

<br>
Cheers and Happy coding!👏 💻 🍷 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**